src/misc/appPackager.ts

```diff
@@ -49,10 +49,12 @@ export class AppPackager {
     }
 
     private overwriteAppManifest(): void {
-        this.fd.info.implements = this.compiledApp.getImplemented()
-            .filter((interfaceName) => !!AppInterface[interfaceName as AppInterface]) as Array<AppInterface>;
+        const i = this.fd.info as any;
 
-        fs.writeFileSync(this.fd.infoFile, JSON.stringify(this.fd.info, null, 4));
+        i.implements = this.compiledApp.getImplemented().filter((interfaceName) => !!AppInterface[interfaceName as AppInterface]) as Array<AppInterface>;
+        this.fd.info = i;
+
+        fs.writeFileSync(this.fd.infoFile, JSON.stringify(i, null, 4));
```

Comment:
These changes don't seem to modify the behavior, right?

Reply:
Correct, it just failed to compile until I did this

Reply:
It failed to compile the apps? But this block doesn't handle any compilation

Reply:
Oh, you mean it failed to compile the apps-compiler? Weird, what error did you get? I didn't get any

Reply:
It failed to compile the apps-compiler locally which was weird. It was saying that `implements` was not part of the definition, which it wasn't. So, I'm not sure how it got through 😬

Reply:
Ah! Might've been a problem with the Apps-Engine version. I've updated it to the latest alpha now, should be gone